src/views/RequestView.py

```diff
@@ -10,8 +11,12 @@
 
 @request_api.route('/', methods=['POST'])
 def create():
+  
   try: 
     req_data = request.get_json()
+    # get number of campaign requests
+    campaign = Campaign.get_one_campaign(req_data['campaign_id'])
+    req_data['eth_id'] = len(campaign.requests)
     data = request_schema.load(req_data)
```

Comment:
This is the meat of it. When creating request the FE will send data that includes that campaign_id. get that campaign and determine the number of requests-- that length will be its eth_id. Modify the given data for eth_id and add it to the request data

Reply:
We don't need to do 'eth_id' = len(campaign.requests) + 1? Does 'eth_id' start counting at 0 or 1 on the blockchain?

Reply:
I think Jack said it starts at 0

Reply:
Cool. That makes sense then. Thanks